### ISSUE TITLE
Centralize development dependencies and scripts into an own repo/npm package

### ISSUE THREAD

**Issue by Apollon77** [ioBroker/Community#21]
**Status:** Idea
**Owner:** @AlCalzone @UncleSamSwiss 
**Link:** https://github.com/ioBroker/adapter-dev
**Documentation:** TBD

**Description:**
We have several dev-dependencies and also scripts like "the gulpfile" which is spread into every adapter repo and sorun out of sync very easy.

Idea is to centralize all that stuff into one repo and offer as a npm package
**Next steps:**
...

**Comment by UncleSamSwiss** [comment]
Repo is set up, but does nothing at the moment:
https://github.com/ioBroker/adapter-dev

**Comment by AlCalzone** [comment]
> Next steps:

- shared ESLint config? Separate for JS/TS

**Comment by UncleSamSwiss** [comment]
> * shared ESLint config? Separate for JS/TS

Good idea, please create an issue in the repo (including how we can easily integrate it in adapter repos).
